auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -358,14 +375,33 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
         }
       }
     }
-    const filingResponse = await this.createNamedBusiness(filingBody)
+    return this.createNamedBusiness(filingBody, business.businessIdentifier)
+  }
 
-    if (filingResponse?.errorMsg) {
-      this.$emit('add-unknown-error')
-      return ''
-    } else {
-      return filingResponse.data.filing.business.identifier
+  private async createBusinessRegistration (business: Business): Promise<any> {
+    const filingBody: RegistrationRequest = {
+      filing: {
+        header: {
+          name: FilingTypes.REGISTRATION,
+          accountId: this.currentOrganization.id
+        },
+        business: {
+          legalType: business.nameRequest.legalType,
+          natureOfBusiness: business.nameRequest.natureOfBusiness
```

Comment:
We may or may not need this, right?

I believe this is user-entered text, whereas in grooming today we talked about users entering a NAICS code and we'd look up the NOB. (It would still be text but we could potentially do a reverse lookup on it, whereas a reverse lookup on user-entered text would probably fail.)

I'm just wondering if we need to pre-populate this in the draft. Probably can't hurt though...

Reply:
True. I included this since it is mentioned in Schema.

Reply:
Just thinking ahead a bit... What will our Create UI code do with this pre-populated text? Will the UI prompt for a NAICS code if this text is pre-populated? Will we store the NAICS code or just the NOB?

It may be that leaving this empty is preferable, but I'm not really sure.

If we want the initial Registration draft filing to have an empty NOB, who will clear it? The Filings UI? Now that would be complex (Auth Web creates draft, Filings UI clear a field, Create UI resumes it).

Can you shed any light on this?

Reply:
Yes NOB will be populated from the NR if available but we are not using it for now